schemas/comment.schema.server.js

```diff
@@ -1,10 +1,22 @@
-const mongoose = require('mongoose')
+const mongoose = require('mongoose');
 
 const commentSchema = mongoose.Schema({
     comment_id: String,
+    story_id:String,
```

Comment:
why do we need story_id in comments?

Reply:
It would be convenient for getting the story it belongs to

Reply:
but it is cyclic, we are already storing comments inside story, having story_id again for convenience isn't the right thing according to me.